staging/src/k8s.io/client-go/tools/clientcmd/client_config.go

```diff
@@ -561,7 +561,7 @@ func (config *inClusterClientConfig) ClientConfig() (*restclient.Config, error)
 		return nil, err
 	}
 
-	// in-cluster configs only takes a host, token, or CA file
+	// in-cluster configs only takes a host, token, CA file, or impersonation configurations
```

Comment:
this comment isn't really accurate... impersonation configuration isn't part of in-cluster configs...